scripts/module-path.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# TODO: remove the need for this script ASAP
+MODULE_VERSION=""
+if [[ ! "${VERSION}" =~ 1\..* ]]; then
+    if [[ "${VERSION}" =~ [0-9]+ ]]; then
```

Comment:
no action needed for this PR: this is lots of bash magic 🪄 🤔

I am wondering if tools like https://github.com/magefile/mage could make our lifes easier here in the future 🤔 Most notably here I am missing a unit test asserting these evaluations work correctly.

Reply:
I tested manually:
```bash
% VERSION=1.9.9 ./scripts/module-path.sh 
github.com/mongodb/mongodb-atlas-kubernetes
jose.vazquez@M-LFJJHGWV6N mongodb-atlas-kubernetes % VERSION=2.9.9 ./scripts/module-path.sh
github.com/mongodb/mongodb-atlas-kubernetes/v2
jose.vazquez@M-LFJJHGWV6N mongodb-atlas-kubernetes % VERSION=3.9.9 ./scripts/module-path.sh
github.com/mongodb/mongodb-atlas-kubernetes/v3
jose.vazquez@M-LFJJHGWV6N mongodb-atlas-kubernetes % VERSION=4.9.9 ./scripts/module-path.sh
github.com/mongodb/mongodb-atlas-kubernetes/v4
```

Note this script is expected to go away soon, as I will make a follow up to remove the need to upgrade the makefile for older builds. Instead each version will build as it used to do and the fork will be done as alternate CI jobs.

Also about [mage](https://github.com/magefile/mage), I think it will be hard to replace make with mage at this point in the project. And if introduced as a complement, it might be adding more complexity for little gain. It would have been different if introduced since the beginning so it could replace all makefile AND scripts.